types/systemjs/index.d.ts

```diff
@@ -305,7 +310,6 @@ declare namespace SystemJSLoader {
          * Promise resolves to the module value.
          */
         import(moduleName: string, normalizedParentName?: string): Promise<any>;
```

Comment:
Same like a `get` method, please add a missing generic value.

Reply:
All uses of generics were raising a dtslint error. Since the generics are not associated with either the parameters or the return values of the declaring functions, they essentially behave like a type assertion.

The rule is explained here. https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md